api/Notification.json

```
@@ -650,12 +650,10 @@
             },
             "opera_android": "mirror",
             "safari": {
-              "version_added": "11"
-            },
-            "safari_ios": {
-              "version_added": "16.4",
-              "notes": "Notifications are supported in web apps saved to the home screen."
+              "version_added": false,
+              "notes": "While Safari supports setting the icon property, the browser does not display the icon in the notification."
```

I would suggest to keep this simple, and not to explain the feature. Also, we could mention [this bug](https://bugs.webkit.org/show_bug.cgi?id=280162).

```suggestion
              "impl_url": "https://webkit.org/b/280162",
              "notes": "The property can be set, but has no effect."
```